kokoro/ubuntu/continuous.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Actually we can get rid of `build.sh` and add the same logic into `continuous.sh`, and move `continuous.sh` up to the `kokoro` directory (and share it between Ubuntu and Mac)